components/kyma-environment-broker/internal/storage/tests_utils.go

```diff
@@ -291,43 +188,44 @@ func SetupTestDBTables(connectionURL string) (cleanupFunc func(), err error) {
 }
 
 func isDockerTestNetworkPresent(ctx context.Context) (bool, error) {
-	netReq := testcontainers.NetworkRequest{
-		Name:   DockerUserNetwork,
-		Driver: "bridge",
-	}
-	provider, err := testcontainers.NewDockerProvider()
 
-	if err != nil || provider == nil {
-		return false, errors.Wrap(err, "Failed to use Docker provider to access network information")
+	cli, err := client.NewClientWithOpts(client.FromEnv)
+	if err != nil {
+		panic(err)
 	}
 
-	_, err = provider.GetNetwork(ctx, netReq)
+	filters := filters.NewArgs()
+	filters.Add("name", DockerUserNetwork)
+	filters.Add("driver", "bridge")
+	list, err := cli.NetworkList(context.Background(), types.NetworkListOptions{Filters: filters})
 
 	if err == nil {
-		return true, nil
+		return len(list) == 1, nil
 	}
 
 	return false, nil
```

Comment:
nit: I realize it's how it was implemented originally but given you are refactoring and improving this method, would it make sense to also return the error rather than continue swallowing it?